kifitos.module

```diff
@@ -3,11 +3,10 @@
 use Drupal\kifitos\AgreementConfigInterface;
 
 function kifitos_kifitos_config_presave(AgreementConfigInterface $config) {
-  Drupal::service('kifitos.agreement_resolver')->cacheConfigs([$config]);
+  Drupal::service('kifitos.agreement_resolver')->cacheSingleConfig($config);
 }
 
-function kifitos_rebuild() {
```

Comment:
Since the kifi_config cache will get created  when  requested, I removed the kifitos_rebuild entirely.